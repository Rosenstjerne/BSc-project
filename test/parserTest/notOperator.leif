var int a,b,c;
a = 5;
b = 10;

print(not a == b); # true

