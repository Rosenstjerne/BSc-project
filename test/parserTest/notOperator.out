function null main() {
	var int a, b, c;
	
	a = 5;
	b = 10;
	c = 5;
	print((! (a == b)));
	
}
